packages/frontend/src/main/lib/core/composables/dom.ts

```diff
@@ -72,7 +72,10 @@ export function useNavigationDrawerAutoResize(params: {
     document.body.style.cursor = ''
     navWidth.value = el.style.width
     document.removeEventListener('mousemove', onMouseMove, false)
-    setTimeout(() => eventHub.$emit('resize-viewer'), 300)
+    setTimeout(() => {
+      // @Dim: Why are we resizing the viewer here? We generally want to avoid needless resizes
```

Comment:
it's emitted when the left navigation panel is squeezed around, so we need it :)

Reply:
For actual real resizing it's perfectly fine, but what happens there, is that whenever you click any button on the bottom-mid control panel (toggle section plane, light controls, help, etc) a resize is triggered. That's the issue with this.